src/components/SlateEditor/plugins/embed/types.ts

```diff
@@ -6,4 +6,4 @@
  *
  */
 
-export const TYPE_EMBED = 'embed';
+export const TYPE_NDLA_EMBED = 'ndlaembed';
```

Comment:
Hadde forsåvidt ikkje trengt å endre navnet på typen. Det hadde spart deg for en del endringer.

Reply:
Fikk streng beskjed av hinrek å endre :smile: